### PR TITLE
Add testing for JSD metrics

### PR DESCRIPTION
**Pull request by sjh120**
## Description

For the JSD testing module, it includes aspects such as value range, divergence being 0 for identical data, and the symmetry of JSD
## Types of changes

- [ ] Maintenance (no change in code, maintain the project's CI, docs, etc.)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: tests/metrics/test_jsd.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+synthetic_data_cotinuous = pd.DataFrame(
+    {
+        "col1": [1, 2, 2, 3, 3],
+        "col2": [4, 5, 5, 6, 6],
+    }
+)
+
+# 创建 JSD 实例
+jsd_instance = JSD()
+
```
Comment (Wh1isper) [inline tests/metrics/test_jsd.py:38]: Can we make this as a `pytest.fixture` or parametrize?

**Review by Wh1isper — CHANGES_REQUESTED** [review]
You need to add `from __future__ import annotations` to support new type hint in python3.8/3.9

> from __future__ import annotations was previously scheduled to become mandatory in Python 3.10, but the Python Steering Council twice decided to delay the change ([announcement for Python 3.10](https://mail.python.org/archives/list/python-dev@python.org/message/CLVXXPQ2T2LQ5MP2Y53VVQFCXYWQJHKZ/); [announcement for Python 3.11](https://mail.python.org/archives/list/python-dev@python.org/message/VIZEBX5EYMSYIJNDBF6DMUMZOCWHARSO/)). No final decision has been made yet. See also [PEP 563](https://peps.python.org/pep-0563/) and [PEP 649](https://peps.python.org/pep-0649/).

File: tests/metrics/test_jsd.py (hunk 1 above)
Comment (MooooCat) [reply tests/metrics/test_jsd.py:38]: @sjh120  `pytest.fixture` should be OK, please refer to other test case in `test/`, such as `tests/models/test_ctgan.py`.

**Review by MooooCat — APPROVED** [review]
LGTM

**Pull request merged**